cmd/exchange_wrapper_standards/exchange_wrapper_standards_test.go

```diff
@@ -641,6 +643,9 @@ var acceptableErrors = []error{
 	order.ErrCannotValidateAsset,         // Is thrown when attempting to get order limits from an asset that is not yet loaded
 	order.ErrCannotValidateBaseCurrency,  // Is thrown when attempting to get order limits from an base currency that is not yet loaded
 	order.ErrCannotValidateQuoteCurrency, // Is thrown when attempting to get order limits from an quote currency that is not yet loaded
+	account.ErrExchangeHoldingsNotFound,
+	ticker.ErrNoTickerFound,
```

Comment:
Super Nit:
```suggestion
	ticker.ErrTickerNotFound,
```
Because *NotFound is idiomatic and ErrTicker* (Invalid, NotFound, SmellsWeird) is completey